### DISCUSSION TITLE
Error editor.Parser.parseCss

### DISCUSSION THREAD

**Discussion by andcmatias** [GrapesJS/grapesjs#5361] [General]
On set media query

editor.Parser.parseCss(`@media only screen and (max-width: 600px) {
  .btn {
    background-color: lightblue;
  }
}`)

grapes.min.js:2 Uncaught TypeError: i is not a function
    at Object.parse (grapes.min.js:2:618584)
    at e.parseCss (grapes.min.js:2:620024)
    at <anonymous>:1:15

**Comment by artf** [comment]
I don't see any issue in the latest version
